packages/x-tree-view/src/internals/plugins/useTreeViewLazyLoading/useTreeViewLazyLoading.types.ts

```diff
@@ -46,7 +46,7 @@ export interface UseTreeViewLazyLoadingInstance extends UseTreeViewLazyLoadingPu
    * @param {TreeViewItemId[]} parentIds The ids of the items to fetch the children of.
    * @returns {Promise<void>} The promise resolved when the items are fetched.
    */
-  fetchItems: (parentIds?: TreeViewItemId[]) => Promise<void>;
+  fetchItems: (parentIds: TreeViewItemId[]) => Promise<void>;
```

Comment:
Fetching the root items is now done with `fetchItemChildren` instead of `fetchItems`
This makes the goal of each method clearer.

- `fetchItems` takes a list of items and fetches their children
- `fetchItemChildren` takes a single id and fetch its children (if the id is null then it represents the root, like for all the other methods)